pages/src/benefit-component/benefit-component.less

```diff
@@ -0,0 +1,65 @@
+@import (reference) "../common/variables.less";
+
+.benefit {
+  color: #fff;
```

Comment:
Just a question, isn't there a way to define colors through variables? After all, I've heard there will be light and dark themes. And the case of specifying a specific value in such a case will play a nasty trick.

Reply:
I think we can put it in this variable file `src/common/variables.less`

Reply:
Text color should be cascading, so we will need to have it defined on the parent container ideally.